### PR TITLE
Combine article queries

### PR DESCRIPTION
**Pull request by jonathan-best**
Merge 5 separate queries into two to prevent refilering

### CODE REVIEW

**Comment by radiodario** [comment]
good stuff!

File: src/server/formatters/Articles.js [hunk 1]
```diff
@@ -90,6 +90,10 @@ function formatFilteredAggregation(name, data, replacement) {
   return format(data.aggregations[name].filtered, replacement)
 }
 
+function formatAggregationWithPath(data, replacement) {
+  return format(data, replacement)
+}
+
```
Comment (peter-mouland) [inline src/server/formatters/Articles.js]: looks like you should be calling directly `format` and remove the function `formatAggregationWithPath`??

File: src/server/formatters/Articles.js [hunk 2]
```diff
@@ -54,8 +54,8 @@ export default function formatData(data) {
         user_cohort : formatAggregation('user_cohort', articleData),
         rfv_cluster : formatAggregation('rfv_cluster', articleData),
         is_first_visit : formatAggregation('is_first_visit', articleData),
-        internal_referrer_urls: formatFilteredAggregation('internal_referrer_urls', articleData, 'Not Available'),
-        internal_referrer_types: formatFilteredAggregation('internal_referrer_types', articleData),
+        internal_referrer_urls: formatAggregationWithPath(articleData.aggregations.internal_referrer.urls, articleData, 'Not Available'),
+          internal_referrer_types: formatAggregationWithPath(articleData.aggregations.internal_referrer.types, articleData),
```
Comment (peter-mouland) [inline src/server/formatters/Articles.js]: should all these just call `format`?

**Comment by peter-mouland** [comment]
@jonathan-best do you mind taking us through the change in a demo? ...can't visualise the bug being fixed here!

**Comment by radiodario** [comment]
@peter-mouland it's not a bugfix - jon is nesting the aggregations so that filtering only happens once for all referral aggregations, rather than once per aggregation.

**Comment by peter-mouland** [comment]
ah - got you. cool

**Pull request merged**